src/styles/components/_specification-block.scss

```diff
@@ -13,15 +13,18 @@
     }
 
     @supports not (grid-template-rows: masonry) {
+      grid-template-rows: repeat(20, auto);
```

Comment:
I think the repeating 20 is causing there to be a TON of black, empty space after the specification blocks on each page. 

<img width="1353" alt="Screen Shot 2022-12-06 at 1 12 06 PM" src="https://user-images.githubusercontent.com/69602589/205989878-b7ee6c09-98ae-4751-a336-bf950c87ae18.png">

Can we just leave this as `grid-template-rows: auto`?

Reply:
Thanks, am working on that now! :)